addons/filebeat/README.md

```diff
@@ -0,0 +1,20 @@
+# Filebeat Helm Chart
+Filebeat is a logging agent installed on the machine generating the log files, tailing them, and forwarding the data to either Logstash for more advanced processing or directly into Elasticsearch for indexing.
+
+## Installation
+Below terraform script shows how to use Filebeat Terraform Addon, A complete example is also given [here](https://github.com/clouddrove/terraform-helm-eks-addons/blob/master/_examples/complete/main.tf).
+
+```hcl
+module "addons" {
+  source  = "clouddrove/eks-addons/aws"
+  version = "0.0.4"
+  
+  depends_on       = [module.eks.cluster_id]
+  eks_cluster_name = module.eks.cluster_name
+
+  filebeat               = true
```

Comment:
- Update depends_on to `depends_on = [module.eks]`
- Update `version` to `0.1.0` as this will be available to use in future release.

Reply:
Done